support-files/mysql.spec.sh

```diff
@@ -124,104 +120,42 @@
 %endif
 
 # ----------------------------------------------------------------------------
-# Packager
+# Distribution support
 # ----------------------------------------------------------------------------
-%if %{undefined mysql_packager}
-%define mysql_packager MySQL Build Team <build@mysql.com>
+
+# Disable post build checks for time being.
+BuildConflicts: post-build-checks rpmlint
+
+BuildRequires: gcc-c++ ncurses-devel perl zlib-devel cmake libaio-devel bison flex
+
+%if 0%{?rhel} == 6 || 0%{?rhel} == 7 || 0%{?fedora} == 20 || 0%{?fedora} == 21
+BuildRequires: time
 %endif
 
-# ----------------------------------------------------------------------------
-# Distribution support
-# ----------------------------------------------------------------------------
-%if %{undefined distro_specific}
-%define distro_specific 0
+%if 0%{?suse_version}
+%if 0%{?suse_version} == 1110
+BuildRequires: gdbm-devel gperf openldap2-client procps pwdutils
 %endif
-%if %{distro_specific}
```

Comment:
The reasons to drop this block:
* OBS could not parse BuildRequires fields properly because the required packages were passed via macro expansion
* IIRC similar issue happened with Requires field
* I wanted to replace distro_releasetag with "standard" %dist macro provided by build platforms (not sure if this was a good decision though)
After this only distro_description was left behind, in complicated deeply nested if/else structure (too shame that there is no %ifelse block, would make this much more readable), and I didn't see any reason to keep it.